### ISSUE TITLE
Another approach if you're interested....

### ISSUE THREAD

**Issue by PySimpleGUI** [SirPrexes/Courser#1]
Noticed a post looking for help updating once a second.

I don't have a Qt answer for you...sorry... but I do have a short PySimpleGUI program that will do what you're after in a tiny amount of code.

Totally understand if it's not of interest.... sorry if I overstepped any lines in offering it up.... simply wanted to help if possible.  

Wishing you the best of luck in your programming journey....

![image](https://user-images.githubusercontent.com/46163555/164893261-ce5251b0-16c4-4a3b-866d-928b919eee4f.png)

```python
import PySimpleGUI as sg

layout = [  [sg.Text('Track the mouse')],
            [sg.Multiline(size=(40,10), reroute_stdout=True, key='-ML-')],
            [sg.Button('Exit')]  ]

window = sg.Window('Mouse Locator', layout)

while True:
    event, values = window.read(timeout=1000)
    # print(event, values)
    if event == sg.WIN_CLOSED or event == 'Exit':
        break
    print(window.mouse_location())

window.close()

```

**Comment by PySimpleGUI** [comment]
Oh... and this one will update the window... sorry I missed that you don't want to print but instead update the window directly

```python
import PySimpleGUI as sg

layout = [  [sg.Text('Track the mouse')],
            [sg.Text(key='-OUT-')],
            [sg.Button('Exit')]  ]

window = sg.Window('Mouse Locator', layout)

while True:
    event, values = window.read(timeout=1000)
    # print(event, values)
    if event == sg.WIN_CLOSED or event == 'Exit':
        break
    window['-OUT-'].update(f'Mouse location: {window.mouse_location()}')
window.close()

```

Here's a video for ya... note that I clicked on my desktop to show that the window does not need to have focus for this to work.  You'll see the text in the titlebar change when it loses focus.

![explorer_E6k4nC1qtE](https://user-images.githubusercontent.com/46163555/164893402-d7c74f4e-1678-477d-9392-4fdccfd07a06.gif)

**Comment by SirPrexes** [comment]
Yes! thank you!, this is exactly what I was looking for. I chose QT earlier because there was a drag and drop editor and I wanted to make things easy for myself. But as it is always the case, you make everything much more complicated. 

Coding in general is still pretty new to me, but I'm impressed what you can do and what tutorials are available. I even managed to add a color field next to the X and Y coordinates that shows me the color of the pixel. 

Now I'm trying to get the X and Y coordinates displayed again at the bottom of the program when I press a certain key. 

You really helped me a lot :)

**Comment by SirPrexes** [comment]
https://user-images.githubusercontent.com/104094724/164915230-b1e93cf2-1036-400d-80a0-d17cc9f1af05.mp4

**Comment by PySimpleGUI** [comment]
You're quite welcome!

![image](https://user-images.githubusercontent.com/46163555/164916812-9a4ce776-a796-4bee-91d9-6054a6632b61.png)

Your program is amazing!

![image](https://user-images.githubusercontent.com/46163555/164916851-0bea0c7a-fbf5-42d8-a84f-1cf5c55afe70.png)

Love what you've done with the pixel capture!  Way to go!!

IMO, you've got the exact right way dialed in to learn to program..... you are building things you dream up.  This is what computer science is all about.... solving problems, building your dreams, making sh*t!  I'm impressed with how quickly you took my example and expanded it.  

Glad I was able to help in some way.  Keep on building!!
![image](https://user-images.githubusercontent.com/46163555/164916938-d0abb65f-7659-4962-9704-bfc8861f0e9a.png)

**Comment by PySimpleGUI** [comment]
Post the code and a screenshot in your readme!  I'm sure, _**sure**_, other people will like what you've made.

**Comment by PySimpleGUI** [comment]
> you make everything much more complicated

That's an awfully wise statement from a beginner.
